pandas/tests/tseries/test_timezones.py

```diff
@@ -1767,3 +1767,22 @@ def compare_local_to_utc(tz_didx, utc_didx):
                                   tslib.maybe_get_tz('Asia/Tokyo'))
         tm.assert_numpy_array_equal(result, np.array(
             [tslib.iNaT], dtype=np.int64))
+
+
+if sys.platform == 'win32':
+    try:
+        import win32timezone
+        import pywintypes
+
+        class TestWin32TimeZone(tm.TestCase):
+
+            def test_win32_tz(self):
+                tz = win32timezone.TimeZoneInfo('Eastern Standard Time')
+                wt = pywintypes.TimeType(2008, 5, 12, 9, 50, tzinfo=tz)
+                tt = Timestamp(wt)
```

Comment:
No, it now falls through and gets the timezone correctly because of the NotImplementedError. That's raised when trying to get the fixed offset. It does however log that it's ignoring that error, which given that it's now working makes me think that actually raising that exception isn't the best thing to do.

I'll close this PR for now and re-open once I've had a chance to look at the best way to get this to work without logging warnings, and taking your feeback re the test cases into consideration.